uaclient/entitlements/__init__.py

```diff
@@ -38,12 +41,15 @@ def valid_services(allow_beta: bool = False) -> List[str]:
     allow_beta |= allow_beta_cfg
 
     if allow_beta:
-        return sorted(ENTITLEMENT_CLASS_BY_NAME.keys())
-
-    return sorted(
-        [
-            ent_name
-            for ent_name, ent_cls in ENTITLEMENT_CLASS_BY_NAME.items()
-            if not ent_cls.is_beta
-        ]
-    )
+        entitlements = ENTITLEMENT_CLASSES
+    else:
+        entitlements = [entitlement for entitlement in ENTITLEMENT_CLASSES if not entitlement.is_beta]
+
+    if all_names:
```

Comment:
What situation we would need all names to be returned ?

Reply:
In cases we are interested in any valid service name, not just the presented one - the line on postinst for instance would need visibility to `cis` even in newer series